code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -54,6 +69,7 @@ const CourseProgress = React.createClass({
             )}
           </div>
         )}
+        {peerReviewSection}
```

Comment:
A pattern @Bjvanminnen suggested is putting all the JSX inside the `return (...)` of `render()` rather than using local variables.